plenum/server/node.py

```diff
@@ -499,21 +497,19 @@ def getPrimaryStorage(self):
         This is usually an implementation of Ledger
         """
         if self.config.primaryStorage is None:
-            fields = getTxnOrderedFields()
-            defaultTxnFile = os.path.join(self.config.baseDir,
+            genesis_txn_initiator = GenesisTxnInitiatorFromFile(self.config.baseDir,
                                        self.config.domainTransactionsFile)
-            if not os.path.exists(defaultTxnFile):
+            if not os.path.exists(genesis_txn_initiator.init_file):
```

Comment:
Yet another places with `if not os.path.exists(genesis_txn_initiator.init_file)`